x-pack/plugins/apm/public/components/app/ServiceOverview/NoServicesMessage.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiEmptyPrompt } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { KibanaLink } from '../../shared/Links/KibanaLink';
+import { SetupInstructionsLink } from '../../shared/Links/SetupInstructionsLink';
+
+interface Props {
+  hasDataEver: boolean;
+}
+
+export function NoServicesMessage({ hasDataEver }: Props) {
+  if (hasDataEver) {
+    return (
+      <EuiEmptyPrompt
+        title={
+          <div>
+            {i18n.translate('xpack.apm.servicesTable.notFoundLabel', {
+              defaultMessage: 'No services found'
+            })}
+          </div>
+        }
+        titleSize="s"
+      />
+    );
+  } else {
+    return (
+      <EuiEmptyPrompt
+        title={
+          <div>
+            {i18n.translate('xpack.apm.servicesTable.noServicesLabel', {
+              defaultMessage: `Looks like you don't have any APM services installed. Let's add some!`
+            })}
+          </div>
+        }
+        titleSize="s"
+        body={
+          <React.Fragment>
+            <p>
```

Comment:
Question: should we use `EuiText` here over `p`?

Reply:
I thought about that actually, but saw that the `EuiEmptyPrompt` example in their docs uses `<p>` tags, so I'm not sure?

https://elastic.github.io/eui/#/display/emptyprompt

Reply:
Looks like the empty prompt component wraps everything in an `EuiTextColor` component under the hood:

https://github.com/elastic/eui/blob/master/src/components/empty_prompt/empty_prompt.js#L63-L67

Reply:
I'm fine with `p`